src/coreclr/jit/optimizebools.cpp

```diff
@@ -29,6 +29,16 @@ struct OptTestInfo
     bool       isBool;   // If the compTree is boolean expression
 };
 
+struct IntBoolOpDsc
```

Comment:
Instead of the struct-and-initializer pattern, could you please rewrite this as a class so that all the relevant transformations are scoped to it? `OptBoolsDsc` might be a useful model.